lib/request.js

```diff
@@ -292,8 +291,7 @@ module.exports = {
    */
 
   get idempotent() {
-    const methods = ['GET', 'HEAD', 'PUT', 'DELETE', 'OPTIONS', 'TRACE'];
-    return !!~methods.indexOf(this.method);
+    return ['GET', 'HEAD', 'PUT', 'DELETE', 'OPTIONS', 'TRACE'].includes(this.method);
```

Comment:
@iyuq You can even write something more, like `return /^[GHPDOT][ACEILNTPOUR]{1,5}[TDES]$/.test(this.method)`. Not sure if looks nicer hovewer...